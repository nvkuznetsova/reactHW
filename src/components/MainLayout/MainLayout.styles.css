.main-header {
  margin-bottom: 30px;
  padding: 10px;
  box-shadow: 0px 3px 5px rgba(0, 0, 0, 0.25);
}

.main-content {
  margin: 0 30px;
}