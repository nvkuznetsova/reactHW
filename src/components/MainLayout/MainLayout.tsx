import React from 'react';
import { Link } from 'react-router-dom';

import { ROUTES } from 'src/constants/routes';
import './MainLayout.styles.css';

export const MainLayout: React.FC = ({ children }) => (
  <div className="container-fluid">
    <header className="main-header">
      <Link to={ROUTES.cart} className="btn btn-success">Cart</Link>
    </header>
    <section className="main-content">
      {children}
    </section>
  </div>
);