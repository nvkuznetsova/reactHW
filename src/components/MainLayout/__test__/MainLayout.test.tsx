import React from "react";
import { shallow } from "enzyme";

import { MainLayout } from "../MainLayout";

it('should match snapshot', () => {
  const children= <span>Test</span>;
const component = shallow<typeof MainLayout>(<MainLayout>{children}</MainLayout>);

expect(component).toMatchSnapshot()
});