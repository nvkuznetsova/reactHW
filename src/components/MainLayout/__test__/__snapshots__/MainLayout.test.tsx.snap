// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainLayout should match snapshot 1`] = `
<div
  className="container-fluid"
>
  <header
    className="main-header"
  >
    <Link
      className="btn btn-success"
      to="/cart"
    >
      Cart
    </Link>
  </header>
  <section
    className="main-content"
  >
    <span>
      Test
    </span>
  </section>
</div>
`;
